combat-trainer.lic

```diff
@@ -255,10 +260,10 @@ class LootProcess
       check_skinning(DRRoom.dead_npcs.first, game_state)
     end
     while 'and get ready to search it' == bput('loot', 'You search', 'I could not find what you were referring to', 'and get ready to search it')
+      @loot_timer = Time.now
```

Comment:
If a loot occurred successfully, the while loop exits and `@loot_timer` is never updated.

If loot occurs but fails, the while loop pauses and retries. In that case I do not think we would want to update `@loot_timer`.